invenio_communities/config.py

```diff
@@ -25,7 +28,7 @@
     "requests": "/communities/<pid_value>/requests",
     "new_subcommunity": "/communities/<pid_value>/subcommunities/new",
     "settings_privileges": "/communities/<pid_value>/settings/privileges",
-    "settings_curation_policy": "/communities/<pid_value>/settings/curation-policy",
+    "settings_submission_policy": "/communities/<pid_value>/settings/submission-policy",
```

Comment:
are we splitting the review_policy and the record_submission_policy in different tabs?

Reply:
From my understanding after the demo, we wanted to put them together 🤔

Reply:
Sorry I was not in the demo... If there is a common decision then I think it is fine...but after our review with @0einstein0 we thought clearer to split them and not confuse the 2 of them... Maybe we discuss this tomorrow on the standup and if we agree then I think we are close to merging this!